app/src/conditional_layer.c

```diff
@@ -33,13 +33,13 @@ struct conditional_layer_cfg {
     int8_t then_layer;
 };
 
-#define IF_LAYER_BIT(i, n) BIT(DT_PROP_BY_IDX(n, if_layers, i)) |
+#define IF_LAYER_BIT(i, n) BIT(DT_PROP_BY_IDX(n, if_layers, i))
 
 // Evaluates to conditional_layer_cfg struct initializer.
 #define CONDITIONAL_LAYER_DECL(n)                                                                  \
     {                                                                                              \
-        /* TODO: Replace UTIL_LISTIFY with DT_FOREACH_PROP_ELEM after Zepyhr 2.6.0 upgrade. */     \
-        .if_layers_state_mask = UTIL_LISTIFY(DT_PROP_LEN(n, if_layers), IF_LAYER_BIT, n) 0,        \
+        /* TODO: Replace LISTIFY with DT_FOREACH_PROP_ELEM after Zepyhr 2.6.0 upgrade. */     \
```

Comment:
Should we just do this TODO comment now, or is this no longer valid?

Reply:
Still valid. Just did the move. Good catch.